audiomentations/augmentations/spectrogram_transforms.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         assert fill_mode in ("mean", "constant")
         self.fill_mode = fill_mode
         self.fill_constant = fill_constant
+        self.name = "SpecFrequencyMask"
```

Comment:
I believe these names are not necessary now, right? See also SpecChannelShuffle

Reply:
Yes, I forgot to remove them^^